test-automation-feign/src/main/kotlin/com/nortal/test/feign/configuration/FeignProperties.kt

```diff
@@ -0,0 +1,50 @@
+package com.nortal.test.feign.configuration
+
+import org.springframework.boot.context.properties.ConfigurationProperties
+import org.springframework.boot.context.properties.ConstructorBinding
+
+@ConstructorBinding
+@ConfigurationProperties(prefix = "test-automation.feign")
+data class FeignProperties(
+        /**
+         * Sets the default read timeout for new connections.
+         * The read timeout is applied to both the TCP socket and for individual read IO operations.
+         * The default value is 30 seconds.
+         */
+        val readTimeout: Int = 30,
```

Comment:
Since feign expects long values here, define them as that

Reply:
Changed type to long